extensions/sql-migration/src/wizard/integrationRuntimePage.ts

```diff
@@ -680,4 +729,13 @@ export class IntergrationRuntimePage extends MigrationWizardPage {
 			}
 		}
 	}
+
+	private atleastOneNodeOnline(nodes: IntegrationRuntimeNode[]): boolean {
+		let result = false;
+		nodes.forEach(node => {
+			if (node.status === constants.ONLINE)
+				result = true;
+		});
+		return result;
```

Comment:
No change needed here, just adding here for reference. You can do this one line code like,

return nodes.some(node => node.status === constants.ONLINE);

it does the same thing. :)